README.md

```diff
@@ -8,10 +8,9 @@
 
 ## getting started
 
-You'll need **python 3.6** and **Google Chrome**. (*Warning*: Python 3.7 is **not** supported at this time)
+You'll need **python 3.6** and **Google Chrome**. (_Warning_: Python 3.7 is **not** supported at this time)
```

Comment:
Could we make the doc clean up a separate PR instead of including them in the --nan-to-num change? I can do that since I know the git clients for linux for windows don't have the best GUI for extracting line level changes.

Reply:
OK with me.  I understand you are doing this, right?

Reply:
yep!